website/source/partials/builders/_community_builders.html.md

```diff
@@ -1,11 +1,13 @@
 ### Community Builders
 
-- [ARM builder](https://github.com/solo-io/packer-builder-arm-image) - A builder
+- [ARM builder]() - A builder
   for creating ARM images
+  * [packer-builder-arm-image](https://github.com/solo-io/packer-builder-arm-image) - simple builder lets you extend on existing system images.
+  * [packer-builder-arm](https://github.com/mkaczanowski/packer-builder-arm) - flexible builder lets you extend or build images from scratch with variety of options (ie. custom partition table).
```

Comment:
I think this could be something like:
```
- ARM builders
  * [packer-builder-arm-image](https://github.com/solo-io/packer-builder-arm-image) - simple builder lets you extend on existing system images.
  * [packer-builder-arm](https://github.com/mkaczanowski/packer-builder-arm) - flexible builder lets you extend or build images from scratch with variety of options (ie. custom partition table).
```